tools/Docker-dev/environment.yml

```diff
@@ -0,0 +1,11 @@
+name: scipydev 
+dependencies:
+  - pytest=6.2.1
+  - pybind11=2.6.1
+  - compilers=1.1.3
+  - matplotlib=3.3.3
+  - numpy=1.19.5
+  - mypy=0.800
+  - python=3.8
+  - sphinx=3.4.3
+  - cython=0.29.21
```

Comment:
Could you add `pythran=0.9.8.post2` (it's an optional build dependency)

Also worth considering to add:
```
- ipython=7.19.0  # that version has working autocompletion it looks like
```
it's kind of an essential development tool I think. Or maybe it could come from somewhere else, I don't know much about how best to layer the images.

Reply:
Also - would it be worth adding `conda-build`? I noticed the Linux instructions with Conda do recommend using `conda develop scipy` and I could perhaps use this as an initialization command later on for gitpod

Reply:
Yes that makes sense to add.